### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by raghav2110**
Added Ui in react for Login and SignUp page. Please review it.

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
- Semantic HTML tags used. 
- rem units for CSS.
- CSS variables used.

Good to merge

File: src/App.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { Products } from "./pages/Products/ProductPage";
 import { Login } from "./pages/Login/LoginPage";
 import { Wishlist } from "./pages/Wishlist/WishlistPage";
 import { Cart } from "./pages/Cart/CartPage";
+import { SignUp } from "./pages/SignUp/signupPage";
```
Comment (adith-a-danthi) [inline src/App.js:8]: You can use barrel imports here

Comment (adith-a-danthi) [inline src/App.js:8]: You can use barrel imports here.

**Review by adith-a-danthi — COMMENTED** [review]
- `LoginPage.css` and `signup.css` seem to have mostly the same code, they can be replaced with one single CSS file (say `Auth.css`).
- Barrel imports for pages
Apart from that good to merge

**Pull request merged**